src/main/Group.java

```diff
@@ -0,0 +1,22 @@
+package main;
+
+public class Group {
+    private int numberOfGroup;
+    private Student[] students;
+
+    public Student[] getStudents() {
+        return students;
+    }
+
+    public int getNumberOfGroup() {
+        return numberOfGroup;
+    }
+
+    public void setNumberOfGroup(int numberOfGroup) {
+        this.numberOfGroup = numberOfGroup;
+    }
+
+    public void setStudents(Student[] students) {
+        this.students = students;
+    }
+}
```

Comment:
override toString() in classes. It is a recommended practice mentioned in Bloch's Effective Java. To learn more, take a look on baeldung.com/java-tostring